utils/python/CIME/test_scheduler.py

```diff
@@ -114,11 +114,8 @@ def __init__(self, test_names, test_data=None,
                        "Missing baseline comparison directory %s" % full_baseline_dir)
 
             # the following is to assure that the existing generate directory is not overwritten
-            if self._baseline_gen_name and not allow_baseline_overwrite:
+            if self._baseline_gen_name:
                 full_baseline_dir = os.path.join(self._baseline_root, self._baseline_gen_name)
-                expect(not os.path.isdir(full_baseline_dir),
```

Comment:
Should have put this here instead of directly on slack:
Here is what I have now:
```Testnames: ['ERS_D_N2_Ld3.f10_f10.ICLM45.yellowstone_gnu.clm-default', 'NCK_D_Ld3.f10_f10.ICLM45.yellowstone_gnu.clm-default']
Using project from env ACCOUNT: P93300606
ERROR: Baseline directories already exists ['/glade/p/cesmdata/cseg/ccsm_baselines/cesm2jpetests/ERS_D_N2_Ld3.f10_f10.ICLM45.yellowstone_gnu.clm-default', '/glade/p/cesmdata/cseg/ccsm_baselines/cesm2jpetests/NCK_D_Ld3.f10_f10.ICLM45.yellowstone_gnu.clm-default']
Use --allow_baseline_overwrite to avoid this error```